src/tools/Repl/Program.cs

```diff
@@ -409,12 +436,15 @@ private static string PrintResult(FormulaValue value, bool minimal = false)
 
                         foreach (var row in table.Rows)
                         {
-                            column = 0;
-                            resultString += "\n ";
-                            foreach (var field in row.Value.Fields)
+                            if (row.Value != null)
```

Comment:
>if (row.Value != null) [](http://example.com/codeflow?start=28&length=22)

We should have an Else here for row.IsError, row.Isblank.

Reply:
Updated the code. thanks